index.html

```diff
@@ -20,6 +20,7 @@ <h1>Our Time Tracker</h1>
 			<input type="text" name="task-input" class="task-input" placeholder="What are you working on?" />
 			<input type="submit" class="add-task-submit" value="Add Task" />
 		</form>
+			</section>

```

Comment:
This is an inline comment. I just discovered I can make some to your code

Reply:
Pretty cool feature. This is helpful